Assets/tiny_rogue/Scripts/Systems/CreatureMovementSystem.cs

```diff
@@ -12,10 +12,10 @@ public class CreatureMovementSystem : ComponentSystem
 
         protected override void OnUpdate()
         {
-            var gss = EntityManager.World.GetExistingSystem<GameStateSystem>();
```

Comment:
Apart from this one line this is just whitespace changes because the alignment in this file was fucked up